octodns/provider/route53.py

```diff
@@ -29,19 +31,84 @@ def _octal_replace(s):
 class _Route53Record(object):
 
     @classmethod
-    def new(self, provider, record, creating):
```

Comment:
For the most part this shifted down and I broke out some of it's logic into `_new_geo`. `_new_dynamic` should be wholly new code. It bascially creates the various RRSets that make up a dynamic record in Route53.